tests/baselines/reference/user/formik.log

```diff
@@ -0,0 +1,17 @@
+Exit Code: 1
+Standard output:
+index.tsx(30,17): error TS7006: Parameter 'values' implicitly has an 'any' type.
```

Comment:
Not sure what is going on here -- these are variables in the test program that were formerly contextually typed. Did we change JSX contextual types? Did formik drop some important part of their types?

Reply:
Afaik we haven't futzed with jsx in the last few days, so it's probably formik?

Reply:
I'm not so sure. I looked around formik's commits and didn't see anything suspicious. And I when I load up the example, it looks like we're incorrectly putting a no-implicit-any error whenever a parameter gets the contextual type any. I can't repro it with non-JSX code, so maybe it's something to do with JSX?

Reply:
`Formik` didn't change - `React` did. Their `LibraryManagedAttributes` impl seems to break `formik`. :(

Reply:
The problem is twofold, but is mostly the inference variant of https://github.com/Microsoft/TypeScript/issues/26274